lib/migration.pm

```diff
@@ -119,7 +119,8 @@ sub disable_installation_repos {
 # Record disk info to help debug diskspace exhausted
 # issue during upgrade
 sub record_disk_info {
-    if (get_var('FILESYSTEM', 'btrfs') =~ /btrfs/) {
+    my $out = script_output 'mount | grep btrfs | grep "on / "';
```

Comment:
I think findmnt is better fit here, but whatever you prefer - code looks ok
```suggestion
    my $out = script_output 'findmnt -n -o fstype /';
```

Reply:
I agree your cmd is better to check btrfs, so I will update code with your cmd and merge with a verify log. Thanks.